code/segmentedtickerplant/datastripe.q

```diff
@@ -22,3 +22,29 @@ initdatastripe:{
      };
 
 if[.ds.datastripe;.proc.addinitlist[(`initdatastripe;`)]];
+
+\d .stpps
+//function to grab where clause from config tables
+segmentfilterfunc:{[tbl;segid] 
+  string first .stpps.segmentfiltermap[exec wcRef from .stpps.segmentconfig where table=tbl , segmentID=segid]};
+// Function to use segmentfilterfunc and return "" incase of error
+segmentfilter:{[tbl;segid]
+  output:.[segmentfilterfunc;
+  (tbl;segid) ;
+  "" ];
```

Comment:
```suggestion
  output:.[segmentfilterfunc;(tbl;segid);""];
```

think this is short enough that it can/should be a single line

Reply:
Thanks, I have now made these changes